### DISCUSSION TITLE
nebula 3.3.0 installation issues with ARM architecture & Java client

### DISCUSSION THREAD

**Discussion by codegladiator007** [vesoft-inc/nebula#4904] [Q&A] [answered]
Hi all - I am facing issues with nebula 3.3.0 installation with ARM architecture with the Java client. I tried commenting out the nebula-console invocation in docker-compose.yml, instead tried running nebula-console locally, with all the ports mapped out, still did not work. Keep getting an error deep within the C code, like so:

./nebula-console/nebula-console -addr IP_ADDR1 -port 9670 -u root -p nebula -e 'ADD HOSTS [IP_ADDR1:9779](IP_ADDR2:9779/),[IP_ADDR3:9779](IP_ADDR3:9779/),[IP_ADDR2:9779](IP_ADDR2:9779/)'

goroutine 1 [running]:
log.Panicf({0x140000780c0?, 0x3e?}, {0x0?, 0x3?, 0x3?})
/usr/local/go/src/log/log.go:395 +0x68
main.main()
CODEPATH/nebula-console/main.go:535](CODEPATH/nebula-console/main.go:535) +0x67c

**Comment by wey-gu** [comment, marked as answer]
Thanks @codegladiator007 this was due to the console docker image now is single arch(AMD64), I am working on that https://github.com/vesoft-inc/nebula-console/pull/189 .

Please ignore this error. you could download binary of console yourself from https://github.com/vesoft-inc/nebula-console/releases (if it's an ARM linux go for https://github.com/vesoft-inc/nebula-console/releases/download/v3.3.1/nebula-console-linux-arm64-v3.3.1 )

Then:

```bash
chmod +x nebula-console-linux-arm64-v3.3.1

./nebula-console-linux-arm64-v3.3.1 -addr localhost -port 9670 -u root -p nebula -e 'ADD HOSTS "storaged0":9779,"storaged1":9779,"storaged2":9779'
```

And afterwards you could access console with

```bash
./nebula-console-linux-arm64-v3.3.1 -addr localhost -port 9670 -u root -p nebula
```
